tests/dialects/test_llvm.py

```diff
@@ -31,16 +32,66 @@ def test_llvm_pointer_ops():
 
     assert isinstance(nullptr.nullptr.typ, llvm.LLVMPointerType)
     assert isinstance(nullptr.nullptr.typ.type, builtin.NoneAttr)
-    assert isinstance(nullptr.nullptr.typ.addr_space, builtin.NoneAttr)
+    assert isinstance(nullptr.nullptr.typ.addr_space, builtin.NoneAttr)    
 
 
 def test_llvm_pointer_type():
     assert llvm.LLVMPointerType.typed(builtin.i64).is_typed()
     assert llvm.LLVMPointerType.typed(builtin.i64).type is builtin.i64
     assert isinstance(
         llvm.LLVMPointerType.typed(builtin.i64).addr_space, builtin.NoneAttr)
-
     assert not llvm.LLVMPointerType.opaque().is_typed()
     assert isinstance(llvm.LLVMPointerType.opaque().type, builtin.NoneAttr)
     assert isinstance(llvm.LLVMPointerType.opaque().addr_space,
                       builtin.NoneAttr)
+    
+    
+def test_llvm_getelementptr_op_invalid_construction():
+    size = arith.Constant.from_int_and_width(1, 32)
```

Comment:
I am personally not in favor of this kind of testing as it introduces an unnecessary dependency. If arith.Constant is changed, this completely unrelated test would have to be changed as well.

IMO, it is better style to use `OpResult(i32)` in this kind of case.

Reply:
I don't feel quite as strongly about the two other Operations here, as they are both part of the LLVM Dialect and this is the LLVM tests. Still, I think these could be replaced by OpResults as well.

Reply:
Ah yes, that makes sense. I'll remove the dependency as much as possible.

Reply:
Unfortunately this results in more linting errors, since we're creating an OpResult with invalid arguments. I'd personally rather have the dependency on arith.Constant, and change if need be.

Reply:
If we only had a uniform testing infrastructure. One that is type safe, gives easy ways to generate values, does not introduce these dependencies... :wink:

Reply:
(I am fine to merge with the arith.constant approach)